### DISCUSSION TITLE
How to undo a "Disable rule" quickfix?

### DISCUSSION THREAD

**Discussion by Royzyc** [valentjn/vscode-ltex#235] [Q&A] [answered]
I am new to vscode and latex and I have just come across a problem when using LTeX.

I unintentionally clicked the ''Disable rule'' in the ‘’Quick fix‘’ column, and LTeX stopped reporting a grammar or spelling mistake. I wonder how to enable this function. I have tried to add some codes in setting.json and also input the command: check current document, but they didn't work. 

I want help. Thanks!

**Comment by valentjn** [comment, marked as answer]
See here on where this is saved: https://valentjn.github.io/vscode-ltex/docs/advanced-usage.html#external-setting-files

It depends on whether a workspace has been open or not. The exact locations of the folders can be looked up via the [`LTeX: Show status information`](https://valentjn.github.io/vscode-ltex/docs/commands.html#ltex-show-status-information) command.

**Reply by Royzyc** [reply]
Thank you very much! I have solved my problem with your method.

**Reply by wang-dua** [reply]
> Thank you very much! I have solved my problem with your method.

 sorry , i cant understand what you say, I have encountered this problem, do not know how to recover

**Comment by wang-dua** [comment]
sorry , i cant understand what you say

**Comment by pkufzh** [comment]
I had encountered the same problem... How can I fix it? Thx!

**Reply by wang-dua** [reply]
兄弟, 不知道你已经解决没有, 我当时没看懂答主说的, 但是后来在设置-扩展-LTex设置-里找到了恢复的选项, 你可以找找看

**Comment by pkufzh** [comment]
感谢！我找到了原因，它在文件夹自动创建了一个 disabled rule 的 txt 文件，里面有一个取消拼写检查的命令，删掉就好了！


---- 回复的原邮件 ----
发件人  ***@***.***> 日期  2025年04月14日 21:44 收件人  ***@***.***> 抄送至  Small ***@***.***>***@***.***> 主题  Re: [valentjn/vscode-ltex] How to undo a "Disable rule" quickfix? (Discussion #235)  

兄弟, 不知道你已经解决没有, 我当时没看懂答主说的, 但是后来在设置-扩展-LTex设置-里找到了恢复的选项, 你可以找找看
